src/mina-signer/tests/rosetta.test.ts

```diff
@@ -19,10 +21,46 @@ describe('Rosetta', () => {
     "stake_delegation": null
   }`;
 
+  // NOTE: copied from the test-vectors legacy signatures file
```

Comment:
found the sign legacy unit tests and thought these would make a (better?) test candidate